etl/src/main/java/org/pih/hivmigration/etl/sql/PcrTestsMigrator.groovy

```diff
@@ -0,0 +1,149 @@
+package org.pih.hivmigration.etl.sql
+
+class PcrTestsMigrator extends SqlMigrator {
+
+    @Override
+    def void migrate() {
+
+        executeMysql("Create staging table for migrating HIV PCR TEST", '''
+            create table hivmigration_pcr_tests (                            
+              encounter_id int PRIMARY KEY AUTO_INCREMENT,
+              source_encounter_date date,
+              source_infant_id int,
+              sample_id varchar(255),
+              result varchar(32), 
+              date_of_result date,              
+              comments varchar(255)            
+            );
+        ''')
+
+        setAutoIncrement('hivmigration_pcr_tests', '(select max(encounter_id)+1 from encounter)')
+
+        loadFromOracleToMySql('''
+            INSERT INTO hivmigration_pcr_tests (                
+                source_infant_id,
+                sample_id,
+                source_encounter_date,
+                result, 
+                date_of_result,
+                comments
+            ) VALUES (?, ?, ?, ?, ?, ?)
+        ''',
+                '''
+            select  INFANT_ID, 
+                    SAMPLE_ID, 
+                    case 
+                        when (SAMPLE_DATE is not null ) then to_char(SAMPLE_DATE, 'yyyy-mm-dd') 
+                        else to_char(RESULT_ENTRY_DATE, 'yyyy-mm-dd')  
+                    end as encounter_date,         
+                    lower(RESULT) as result,        
+                    case 
+                        when (DATE_RESULT_RECEIVED is not null ) then to_char(DATE_RESULT_RECEIVED, 'yyyy-mm-dd') 
+                        else to_char(RESULT_ENTRY_DATE, 'yyyy-mm-dd')  
```

Comment:
I think we can just use date_result_received here, and just keep null if there's no date result received (and then don't create an "date of results" obs below for these entries)...